src/AsyncGenerator.yml

```diff
@@ -0,0 +1,249 @@
+﻿solution:
+  filePath: ..\..\..\src\NHibernate.sln #wants to resolve from the execuing path.
```

Comment:
The `AsyncGenerator.Tests` assembly was published by mistake, the problem is that the current yaml configuration [targets it](https://github.com/nhibernate/nhibernate-core/pull/588/files/9cb106e5d7ecf164f8d3638f40d3fd809e622533..8748f86ed0a65c6cc1b94e5d18d3b57be0b49f2f#diff-e4c28185d6771c49374e13aeaefe2bf9R184). For now, we should add the `LinqAsyncCounterpartsFinder` in the [cSharpScript](https://github.com/nhibernate/nhibernate-core/pull/588/files/9cb106e5d7ecf164f8d3638f40d3fd809e622533..8748f86ed0a65c6cc1b94e5d18d3b57be0b49f2f#diff-e4c28185d6771c49374e13aeaefe2bf9R220) property or in a separate assembly. In the following days I will try to make an option to register the extension methods within the configuration.